your-first-cluster.md

```diff
@@ -0,0 +1,512 @@
+  namespace: "default"                   # The is the namespace the Cluster will be created in.
+spec:
+  clusterNetwork:
+    services:                            # The IP range of services in the Cluster.
+      cidrBlocks: ["10.128.0.0/12"]
+    pods:                                # The IP range of pods in the Cluster.
+      cidrBlocks: ["192.168.0.0/16"]
+    serviceDomain: "cluster.local"
+  topology:
+    class: quick-start                   # The name of the ClusterClass the Cluster is templated from.
+    controlPlane:
+      replicas: 1                        # The number of nodes in the Cluster control plane.
+    version: v1.25.0                     # The version of the Kubernetes Cluster.
+    workers:
+      machineDeployments:                # A group of worker nodes that all have the same spec.
+        - class: default-worker          # The name of the Class the MachineDeployment is templated from.
+          name: md-0
+          replicas: 1                    # The number of worker nodes in this MachineDeployment
+```
+
+The Cluster definition above defines some of the fundamental characteristics of our Cluster - it's name, the version of Kubernetes to install, the number of nodes in the control plane, the number and type of worker nodes, as well as some details about networking.
+
+
+Time to create the Cluster! It's as simple as:
+
+```bash
+kubectl apply -f yamls/clusters/docker-cluster-one.yaml
+```
+
+Now that the Cluster has been created we can watch it come into being with:
+```bash=
+clusterctl describe cluster docker-cluster-one
+```
+The output should resemble:
+```bash=
+NAME                                                              READY  SEVERITY  REASON                           SINCE  MESSAGE
+
+Cluster/docker-cluster-one                                        False  Warning   ScalingUp                        84s    Scaling up control plane to 1 replicas (actual
+ 0)
+├─ClusterInfrastructure - DockerCluster/docker-cluster-one-4tqct  True                                              84s
+
+├─ControlPlane - KubeadmControlPlane/docker-cluster-one-bts9k     False  Warning   ScalingUp                        84s    Scaling up control plane to 1 replicas (actual
+ 0)
+│ └─Machine/docker-cluster-one-bts9k-wjkcd                        False  Info	   Bootstrapping                    81s    1 of 2 completed
+
+└─Workers
+
+  └─MachineDeployment/docker-cluster-one-md-0-sfjx8               False  Warning   WaitingForAvailableMachines      84s    Minimum availability requires 1 replicas, curr
+ent 0 available
+    └─Machine/docker-cluster-one-md-0-sfjx8-7dd6cfdbcd-5f6nm	  False  Info	   WaitingForControlPlaneAvailable  84s    0 of 2 completed
+```
+
+Our cluster will also be visible to kind, and we can see it using
+
+```bash
+kind get clusters
+```
+
+To operate on the cluster we need to retrieve its kubeconfig.
+```bash=
+kind get kubeconfig --name docker-cluster-one > cluster-one.kubeconfig
```

Comment:
Same as for mac. Shouldn't work.

I had those notes:
```
# Modify server address with 127.0.0.1 and port from: docker port capi-quickstart-lb 6443/tcp, e.g.
# $ docker port capi-quickstart-lb 6443/tcp
# 0.0.0.0:36293
# kubeconfig: server: 127.0.0.1:36293
```
Not sure if there is a quick & easy powershell snippet